experimental/c-cpp/manager.py

```diff
@@ -900,8 +886,10 @@ def run_introspector_on_dir(build_results, test_dir,
 
 
 def log_fuzzer_source(full_fuzzer_source: str):
-  logger.info('-' * 20, ' HARNESS SOURCE ', '-' * 20, '\n', full_fuzzer_source,
-              '\n', '-' * 56)
+  harness_source_out = '-' * 20 + ' HARNESS SOURCE ' + '-' * 20 + '\n'
+  harness_source_out += full_fuzzer_source + '\n'
+  harness_source_out += '-' * 56
```

Comment:
Could we do something similar to `f'{"-"*20} PROMPT {"-"*20}\n{prompt}\n{'-' * 48}'`?